sim/midas/src/main/scala/midas/widgets/BridgeAnnotations.scala

```diff
@@ -38,7 +38,7 @@ trait BridgeAnnotation extends SingleTargetAnnotation[ModuleTarget] {
   */
 
 case class SerializableBridgeAnnotation[T <: AnyRef](
```

Comment:
Is it possible (really, sensible) to collapse the class hierarchy into `BridgeAnnotation` now?

Reply:
This is addressed in #1224.